docs/getting-started/installation-common/scylla-web-installer.rst

```diff
@@ -32,10 +32,10 @@ options to install a different version or ScyllaDB Enterprise:
      - Description
    * - ``--scylla-product``
      - ``scylla`` | ``scylla-enterprise``
-     - Specifies the ScyllaDB product to install: Open Source (``scylla``) or Enterprise (``scylla-enterprise``)  The default is ``scylla``.
+     - Specifies the product to install: ScyllaDB (``scylla``) or ScyllaDB Enterprise (``scylla-enterprise``)  The default is ``scylla``.
```

Comment:
- what if someone wants to install 2024.1 or 6.2?
- Starting 2025.1 ScyllaDB **is** ScyllaDB Enterprise, so the two alternatives here are not clear to **me**

Reply:
@tzach if 2024.1 is the release they want to install, they should use `--scylla-product scylla-enterprise` , nothing will change here